src/bp-templates/bp-nouveau/includes/xprofile/functions.php

```diff
@@ -43,3 +43,55 @@ function bp_nouveau_xprofile_enqueue_scripts() {
 	}
 	wp_enqueue_script( 'bp-nouveau-xprofile' );
 }
+
+
+/**
+ * Return profile header buttons
+ *
+ * @since BuddyBoss 1.5.1
+ *
+ * @return mixed|void
+ */
+
+function bb_profile_header_buttons() {
+
+	$buttons = apply_filters( 'bb_profile_header_buttons', array(
+		'member_friendship' => __( 'Connect', 'buddyboss' ),
+		'member_follow'     => __( 'Follow', 'buddyboss' ),
+		'private_message'   => __( 'Message', 'buddyboss' ),
+		'member_switch'     => __( 'View As', 'buddyboss' ),
+	) );
+
+	return $buttons;
```

Comment:
@shohelbuddyboss We should not get static buttons array here. Is there any way we can have dynamic buttons for user?

The function name always starts with `bp_` so here it should be `bp_get_profile_header_buttons`

Reply:
@mehulkaklotar I tried to use existing function to get the button, but those buttons not appear at the customizer as buttons appear at the profile header based on various condition. So it's the best way, used filter to add remove buttons here.

Added the prefix bp_ instead of bb_ (bb means buddyboss)